README.md

```diff
@@ -97,10 +97,10 @@ kind delete cluster
 
 <!--TODO(bentheelder): improve this part of the guide-->
 To create a cluster from Kubernetes source:
-- ensure that Kubernetes is cloned in `$(go env GOPATH)/src/k8s.io/kubernetes`
```

Comment:
I think it would be good to keep some mention that the default path is `$(go env GOPATH)/src/k8s.io/kubernetes`, but can be pointed at a different path if provided on the command line.